File: Aulas/Desenvolvimento Web II/2023-2/Aula_01/exercicio3.ts
//Faça um programa que receba três notas, calcule e mostre a média ponderada entre elas.

namespace exercicio3 {
    let nota1: number, nota2: number, nota3: number;
    const peso1: number = 3, peso2: number = 5, peso3: number = 2;

    nota1 = 8, nota2 = 5, nota3 = 4;
    /*if(nota3 == 0){
        situacao = false
    } else {
        situacao = true
    }*/
    let situacao: boolean = nota3 == 0 ? true : false;
    //if(situacao == false)
    if(!situacao && nota2 >= 5){
        let media: number = (nota1 * peso1 + nota2 * peso2 + nota3 * peso3)/(peso1 + peso2 + peso3)
        console.log(`A média final é: ${media}`);
    } else {
        console.log("Aluno não entregou o trabalho e sua média foi 0");
    }

}

File: Aulas/IoT/Codigos/Aula 01 - C/Exercicios/Exercicio3.c
// Faça um programa que receba três notas, calcule e mostre a média ponderada entre ela

#include <stdio.h>

int main() {
    float nota1, nota2, nota3, media;

    printf("Digite as três notas: ");
    scanf("%f%f%f", &nota1, &nota2, &nota3);

    media = (nota1 * 5 + nota2 * 3 + nota3 * 2) / 10;

    printf("A média ponderada é: %.2f\n", media);

    return 0;
}